### PR TITLE
add e2e test cases

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: test/e2e/placement_test.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespace,
+			},
+		}
+		_, err := kubeClient.CoreV1().Namespaces().Create(context.Background(), ns, metav1.CreateOptions{})
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	})
+
+	/*
+		ginkgo.AfterEach(func() {
```
Comment (qiujian16) [inline test/e2e/placement_test.go:47]: clean this up?

**Review by qiujian16 — COMMENTED** [review]
/approve

File: test/e2e/placement_test.go (hunk 1 above)
Comment (elgnay) [reply test/e2e/placement_test.go]: Code updated.

**Pull request merged**